api/src/test/java/io/opentelemetry/OpenTelemetryTest.java

```diff
@@ -371,27 +371,7 @@ public BatchRecorder newMeasureBatchRecorder() {
 
     @Nullable
     @Override
-    public LabelSet createLabelSet(String k1, String v1) {
-      return null;
-    }
-
-    @Nullable
-    @Override
-    public LabelSet createLabelSet(String k1, String v1, String k2, String v2) {
-      return null;
-    }
-
-    @Nullable
-    @Override
-    public LabelSet createLabelSet(
-        String k1, String v1, String k2, String v2, String k3, String v3) {
-      return null;
-    }
-
-    @Nullable
-    @Override
-    public LabelSet createLabelSet(
-        String k1, String v1, String k2, String v2, String k3, String v3, String k4, String v4) {
+    public LabelSet createLabelSet(String... keyValuePairs) {
       return null;
```

Comment:
We should return a NoopLabelSet to not crash if they call record on a metric with the result of this.

Reply:
this is in test code...